ApiConnectorTests/JSONModelNetworkConnectorTests.swift

```diff
@@ -23,7 +23,7 @@ class JSONModelNetworkConnectorTests: XCTestCase {
     func testModelRequest() {
         let postExpactation = expectation(description: "ModelRequestExpectation")
         let post = TestData.defaultPost
-        let postObservable: Observable<Post> = Connection(environment: .test).requestObservable(with: post, at: .pictures)
+        let postObservable: Observable<Post> = Connection(environment: .test).requestObservable(method: .get, with: post, at: .pictures)
```

Comment:
When you add default method param, all these changes should be discarded, as they won't be necessary